modules/custom/express_content_edit_lock/express_content_edit_lock.module

```diff
@@ -247,10 +247,10 @@ function _express_content_edit_lock_recursive(&$element) {
 function express_content_edit_lock_user_login(&$edit, $account) {
   if (module_exists('cu_users')) {
     $devs = cu_users_get_users('developer');
-    $dev_lock = variable_get('lock_user_dev');
+    $dev_lock = variable_get('lock_user_dev', FALSE);
 
-    if (!array_key_exists($account->name, $devs) && ($dev_lock === TRUE)) {
-      watchdog('express_content_edit_lock', t('Site is locked: lock_user_dev'), array($account), WATCHDOG_NOTICE, NULL);
+    if (!array_key_exists($account->name, $devs) && ($dev_lock == 1)) {
+      watchdog('express_content_edit_lock', ':name has tried to login to the site while it is locked.', array(':name' => $account->name));
       user_logout();
```

Comment:
If you take the parts of `user_logout()` then you can set a message.

```php
      user_logout_current_user();
      drupal_set_message('foo bar');
      drupal_goto('user');
```

Reply:
In fact, you could just go to the user page which already has a message about why they can't log in.